engines/real-time-ensembler/requirements.txt

```diff
@@ -0,0 +1,5 @@
+argparse>=1.4.0
+cloudpickle==1.2.2
+orjson==2.6.8
+tornado==6.1
+file:../../sdk
```

Comment:
I think we need to consider moving to versioned sdk packages. I.e., instead of using the latest sources of the Python SDK package, we can add it as a PyPi wheel dependency (`turing-sdk==0.2.0`). 
Maybe this can be done once we fix the contract of the `PyFunc` ensembler.

Reply:
What was the reason why we decided to work with importing the `sdk` directory directly instead of relying on versioned packages?

Reply:
Because there were no `sdk` before we started working on batch ensembling jobs :) Hence, 1) Nothing was published to PyPi at that stage 2) The `sdk` interfaces were fluid and has been changed frequently, together with changes to the `pyfunc-ensembler-job` implementation. So, it made sense to keep them together by making `batch-ensembler` always be dependent on the latest version of the SDK

Reply:
Ah alright, in this case for this PR, I'll leave the dependency on the module of the SDK as it is, i.e. `file:../../sdk` since there's actually a tiny change made to the SDK module in this PR. 

Once this PR has been merged, I'll cut a new release for the SDK, and after that raise a separate PR to update the `requirements.txt` to utilise the version for that latest release. This should permanently move us away from having the actual SDK module in the same repo as a dependency.

Reply:
Cool. Yes, we can definitely move the the PyPi dependency once the changes to SDK are finalised